### DISCUSSION TITLE
"The system cannot find the path specified" on dep ssh <hostname>

### DISCUSSION THREAD

**Discussion by zihadt** [deployphp/deployer#3038] [Help needed]
I am getting the following error on "dep ssh <hostname>" command 

```
#!/usr/bin/env php
The system cannot find the path specified.
The system cannot find the path specified.
```

After some digging I found out the issue occurs in the following line in SshCommand.php on line 89

`        passthru("ssh -t $options {$host->getConnectionString()} 'cd $deployPath/current 2>/dev/null || cd $deployPath; $shell_path'");
`

However if I use only one path in cd command, dep ssh works fine then. Otherwise the line outputs the above mentioned error. 

` passthru("ssh -t $options {$host->getConnectionString()} 'cd $deployPath/current; $shell_path'"); //this works
`

`passthru("ssh -t $options {$host->getConnectionString()} 'cd $deployPath; $shell_path'"); //this also works
`

`        passthru("ssh -t $options {$host->getConnectionString()} 'cd $deployPath/current 2>/dev/null || cd $deployPath; $shell_path'"); //But this fails
`

`passthru("ssh -t $options {$host->getConnectionString()} 'cd $deployPath/current 2>/dev/null; $shell_path'"); //this also fails
`

I was wondering what could be the reason for this weird behavior. 

I am using git bash for Windows on a windows 11 Machine. 
Deployer version 7.0.0 RC-4

**Comment by antonmedv** [comment]
What is your target OS?

**Reply by zihadt** [reply]
Ubuntu 20.04

**Reply by antonmedv** [reply]
What’s shell?  I use same os with bash. All works fine

**Reply by zihadt** [reply]
Me using bash too. Thats why I found it weird and couldn't figure out what's actually causing this.

**Reply by antonmedv** [reply]
Maybe switch only to deploy_path?

**Reply by zihadt** [reply]
Is there any config option for that or should I just modify and use it ?

**Reply by antonmedv** [reply]
I’m going to change code.
